host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@10.0.1.1.cmake

```diff
@@ -75,23 +75,23 @@ set(BLT_CMAKE_IMPLICIT_LINK_DIRECTORIES_EXCLUDE "/usr/tce/packages/gcc/gcc-4.9.3
 
 # Root directory for generated TPLs
 
-set(TPL_ROOT "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2023_02_03_15_43_49/clang-10.0.1.1" CACHE PATH "")
+set(TPL_ROOT "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2023_04_18_15_47_23/clang-10.0.1.1" CACHE PATH "")
 
-set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.8.6" CACHE PATH "")
+set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.8.6-wizqtmncmicwv3wzfzcjuukwmfbqflc2" CACHE PATH "")
```

Comment:
> `-wizqtmncmicwv3wzfzcjuukwmfbqflc2`

Out of curiosity, is this suffix due to a change in our setup or the new spack?

Reply:
its a changed due to setup

was this
https://github.com/LLNL/axom/blob/b27dd885e1a1eddd122ca6f00d2b0d6243ca8832/scripts/spack/configs/config.yaml#L21-L22

now is
https://github.com/LLNL/axom/blob/e534338a186d532ba93c56bb8a79bce18f487175/scripts/spack/devtools_configs/blueos_3_ppc64le_ib_p9/spack.yaml#L10-L11

Reply:
I personally am ambivalent to this change, but if we want to allow other projects to trigger our CI and use spack upstreaming to not rebuild the full set of libraries, it is my understanding it's required.

Think Umpire making changes and triggering our CI to see if it still passes Axom tests.